# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/rene/cdrdao/make-more.diff
# ROCK Linux is Copyright (C) 1998 - 2003 Clifford Wolf
# 
# This program is free software; you can redistribute it and/or modify
# it under the terms of the GNU General Public License as published by
# the Free Software Foundation; either version 2 of the License, or
# (at your option) any later version. A copy of the GNU General Public
# License can be found at Documentation/COPYING.
# 
# Many people helped and are helping developing ROCK Linux. Please
# have a look at http://www.rocklinux.org/ and the Documentation/TEAM
# file for details.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- makefile.org	Sat Dec 28 23:21:18 2002
+++ makefile	Sat Dec 28 23:21:31 2002
@@ -50,10 +50,10 @@
 	@echo antlr executable now in $(BINDIR)
 	(cd ./dlg; $(MAKE) CC="$(CC)" COPT="$(COPT)")
 	@echo dlg executable now in $(BINDIR)
-#	(cd ./sorcerer; $(MAKE) CC="$(CC)" COPT="$(COPT)")
-#	@echo sorcerer executable now in $(BINDIR)
-#	(cd ./support/genmk; $(MAKE) CC="$(CC)" COPT="$(COPT)"; mv genmk ../../$(BINDIR))
-#	@echo genmk executable now in $(BINDIR)
+	(cd ./sorcerer; $(MAKE) CC="$(CC)" COPT="$(COPT)")
+	@echo sorcerer executable now in $(BINDIR)
+	(cd ./support/genmk; $(MAKE) CC="$(CC)" COPT="$(COPT)"; mv genmk ../../$(BINDIR))
+	@echo genmk executable now in $(BINDIR)
 	@echo
 	@echo "       PCCTS 1.33MR32 installation complete"  # MRXXX
 
